deploy/helm/charts/templates/contoller.yaml

```diff
@@ -0,0 +1,128 @@
+          args:
+            - "--v=5"
+            - "--csi-address=$(ADDRESS)"
+            - "--leader-election"
+          env:
+            - name: ADDRESS
+              value: /var/lib/csi/sockets/pluginproxy/csi.sock
+          imagePullPolicy: {{ .Values.controller.resizer.image.pullPolicy }}
+          volumeMounts:
+            - name: socket-dir
+              mountPath: /var/lib/csi/sockets/pluginproxy/
+        - name: {{ .Values.controller.snapshotter.name }}
+          image: "{{ .Values.controller.snapshotter.image.registry }}{{ .Values.controller.snapshotter.image.repository }}:{{ .Values.controller.snapshotter.image.tag }}"
+          imagePullPolicy: {{ .Values.controller.snapshotter.image.pullPolicy }}
+          args:
+            - "--csi-address=$(ADDRESS)"
+            - "--leader-election"
+          env:
+            - name: ADDRESS
+              value: /var/lib/csi/sockets/pluginproxy/csi.sock
+          volumeMounts:
+            - name: socket-dir
+              mountPath: /var/lib/csi/sockets/pluginproxy/
+        - name: {{ .Values.controller.snapshotController.name }}
+          image: "{{ .Values.controller.snapshotController.image.registry }}{{ .Values.controller.snapshotController.image.repository }}:{{ .Values.controller.snapshotController.image.tag }}"
+          args:
+            - "--v=5"
+            - "--leader-election=true"
+          imagePullPolicy: {{ .Values.controller.snapshotController.image.pullPolicy }}
+        - name: {{ .Values.controller.provisioner.name }}
+          image: "{{ .Values.controller.provisioner.image.registry }}{{ .Values.controller.provisioner.image.repository }}:{{ .Values.controller.provisioner.image.tag }}"
+          imagePullPolicy: {{ .Values.controller.provisioner.image.pullPolicy }}
+          args:
+            - "--csi-address=$(ADDRESS)"
+            - "--v=5"
+            - "--feature-gates=Topology=true"
+            - "--strict-topology"
+            - "--enable-leader-election"
+            - "--leader-election-type=leases"
+            - "--extra-create-metadata=true"
+          env:
+            - name: ADDRESS
+              value: /var/lib/csi/sockets/pluginproxy/csi.sock
+          volumeMounts:
+            - name: socket-dir
+              mountPath: /var/lib/csi/sockets/pluginproxy/
+        - name: {{ .Values.zfsPlugin.name }}
+          image: "{{ .Values.zfsPlugin.image.registry }}{{ .Values.zfsPlugin.image.repository }}:{{ .Values.zfsPlugin.image.tag }}"
+          imagePullPolicy: {{ .Values.zfsPlugin.image.pullPolicy }}
+          env:
+            - name: OPENEBS_CONTROLLER_DRIVER
+              value: controller
+            - name: OPENEBS_CSI_ENDPOINT
+              value: unix:///var/lib/csi/sockets/pluginproxy/csi.sock
+            - name: OPENEBS_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
```

Comment:
This is the namespace where ZFS_LocalPV CRs will be created. So we can install it in kube-system namespace and by default openebs was used for creating the CRs. Although this will also work.

Reply:
Since the zfs localpv is getting deployed in kube-system namespace , then the value here will be `kube-system` not `openebs` , is that fine ?

Also i have added the priorityClass in cstor charts PR , PTAL in case we want to add it here ?

Reply:
I would suggest to document this that it will create the resource where we are deployinh the helm chart. In contrast to what zfs operator has where both can be different namespace.